sync_crawler/crawler/cna_crawler.py

```diff
@@ -0,0 +1,38 @@
+from proto.news_pb2 import News
+from sync_crawler.crawler.base_crawler import BaseCrawler
+
+
+class CnaCrawler(BaseCrawler):
+
+    def __init__(self, custom_property, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Remove this bacause python will automatically call `super().__init__()` for you.